core/serv/src/main.rs

```diff
@@ -149,6 +149,7 @@ impl ServiceContext {
             Self::make_entry::<MarketService>(path, "market")?,
             Self::make_entry::<ActivityService>(path, "activity")?,
             Self::make_entry::<PaymentService>(path, "payment")?,
+            Self::make_entry::<PaymentDriverService>(path, "payment")?,
```

Comment:
`payment` twice? Why not `payment-driver`? Or even `gnt-driver`? Each db is payment platform specific.

Reply:
If the idea is that each service should have its own database, then this should be changed.

Reply:
pls change this name to whatever you want. I'm just after splitting it from main `yagna.db`